Tools/CMake/AMReXInstallHelpers.cmake

```diff
@@ -80,20 +80,30 @@ function (install_amrex)
    # Write and install configure file
    include(CMakePackageConfigHelpers)
 
+   #
+   # Setup for install and export of build tree
+   #
+   set(CMAKE_FILES_DIR   lib/cmake/AMReX)   # Relative path to top level installation/build-tree
```

Comment:
for installation, that path is only valid on Unix:
https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure

Reply:
I think one usually sets them via changed defaults in a cached `CMAKE_INSTALL_CMAKEDIR` variable:
https://github.com/openPMD/openPMD-api/blob/ddc673bcd96d28293a11cfdbbc8fdcd57c154385/CMakeLists.txt#L38